### PR TITLE
Audit Item L05

### LINKED ISSUE #26: Resolve audit item L05

### PR DESCRIPTION
**Pull request by rheeger**
closes #26

### CODE REVIEW

**Review by CantelopePeel — APPROVED** [review]
LGTM

File: contracts/Administratable.sol [hunk 1]
```diff
@@ -33,18 +36,14 @@ contract Administratable {
...
     _onlyAdminOrRole(adminContractAddress, role);
     _;
   }
-  
-  function _onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) 
-    private
-    view
-  {
 
+  function _onlyAdminOrRole(address adminContractAddress, IEndaomentAdmin.Role role) private view {
```
Comment (mds1) [inline contracts/Administratable.sol:52]: This function seems to be the only one without docstrings, otherwise this PR looks good!

Comment (CantelopePeel) [reply contracts/Administratable.sol:52]: The docstring would be the same as the modifier above. Robbie do you want to add a brief comment there?

**Review by CantelopePeel — APPROVED** [review]
LGTM

**Review by CantelopePeel — APPROVED** [review]

**Pull request merged**